fitbenchmarking/parsing/fitting_problem.py

```diff
@@ -72,7 +72,7 @@ def __init__(self, options):
         #: e.g.
         #: :code:`[{p1_name: p1_val1, p2_name: p2_val1, ...},
         #: {p1_name: p1_val2, ...}, ...]`
-        self.starting_values = None
+        self.starting_values: list = []
```

Comment:
I'm surprised it's just this one causing problems. I guess the others must be guarded!?